tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -4028,24 +4030,40 @@ TEST_P(OpConverter_FP32_FP16_INT32_Test, ConvertStridedSlice) {
           /*expected_output=*/{1, 2, 4, 5},
           Status::OK(),
           Status::OK(),
-          {-1, 1, 1, 3},
+          /*partial_input_dims=*/{-1, 1, 1, 3},
       },
+      // Invalid parameters: end[2] < begin[2]
+      TestParams{/*input_dims=*/{1, 1, 2, 3},
+                 /*begin=*/{0, 0, 2, 0},
+                 /*end=*/{1, 1, 0, 3},
+                 /*strides=*/{1, 1, 1, 1},
+                 /*begin_mask=*/0,
+                 /*end_mask=*/0,
+                 /*ellipsis_mask=*/0,
+                 /*new_axis_mask=*/0,
+                 /*shrink_axis_mask=*/0,
+                 /*expected_output_dims=*/{},
+                 /*expected_output=*/{},
+                 errors::InvalidArgument("\"size\" cannot be negative for "
+                                         "StridedSlice"),
+                 Status::OK(),
+                 /*partial_input_dims=*/{}},
+      // [Static Input] Support "crop" on the last two dimensions.
```

Comment:
I don't quite understand your comment "crop on the last two dimensions". What I don't understand is that the resulting dimension is (1, 1, 1, 2), why  the 2rd dimension is part of the "crop dimensions" but the 0rd and 1rd dimensions are not part of the "crop dimensions"?

Reply:
The equivalent code is something like the following:
```
tensor = tf.zeros((1, 1, 2, 3))
crop = tensor[0:, 0:, :1, :2]
```
Therefore, the "crop" is only on the last two. I meant "crop" meaning that the dimension slice is not identity for those axes.

Reply:
(1) this following code crop on the last two dimensions, but that is NOT equivalent to the test code here.
tensor = tf.zeros((1, 1, 2, 3))
crop = tensor[:0, :0, :1, :2]

(2) The test code in your PR has "end_mask" and is equivalent to below, which "crop on all four dimensions"
tensor = tf.zeros((1, 1, 2, 3)) 
crop = tensor[0:, 0:, :1, :2]. => same as tensor[:1,:1,:1:2]

Do you agree?

In conclusion, the comment you added is try for